homeassistant/components/tuya/binary_sensor.py

```diff
@@ -348,6 +353,48 @@ async def async_setup_entry(
     """Set up Tuya binary sensor dynamically through Tuya discovery."""
     hass_data = entry.runtime_data
 
+    def _get_fault_labels(device: CustomerDevice) -> list[str]:
```

Comment:
Let's move this out of the method (as in, no in-line method)

Reply:
Done 👍